editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -430,6 +430,8 @@ describe('Use the text editor', () => {
 
         await wait(50) // give it time to adjust the caret position
 
+        typeText('l')
+
         await closeTextEditor()
```

Comment:
can we wrap `await closeTextEditor()` in `expectSingleUndoStep`?

Reply:
I wrapped closeTextEditor-s in expectSingleUndoStep in multiple text editing tests in f48683ce471dad92b2ab71d0e52bda57a9bf5ef2 . Unfortunately, collapsing similar divs still seems to generate more undo steps, but that is not really connected to this PR (text editor still saves only once), so I would not solve it here.